src/core/url_store/atoms/urlStore.ts

```diff
@@ -31,7 +32,16 @@ export const urlStoreAtom = createAtom(
     _setState: (state: UrlData | null) => state,
     init: (initialState: UrlData) => initialState,
   },
-  ({ get, schedule, onAction, onInit, create }, state: UrlData | null = null) => {
+  (
+    { get, schedule, onAction, onInit, create, getUnlistedState },
+    state: UrlData | null = null,
+  ) => {
+    const isFeedSelectorEnabled =
+      typeof (
+        getUnlistedState(featureFlagsAtom)[FeatureFlag.EVENTS_LIST__FEED_SELECTOR] ||
```

Comment:
From what @vkozio told me, it's probably better to use `configRepo.get().features` instead of `featureFlagsAtom`, because it's planned to get rid of `featureFlagsAtom` in the future